src/orquestra/sdk/_base/cli/_dorq/_dumpers.py

```diff
@@ -0,0 +1,89 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+
+"""
+Code that stores values on disk as a result of a CLI command.
+"""
+import json
+import typing as t
+from dataclasses import dataclass
+from functools import singledispatch
+from pathlib import Path
+
+import dill
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Using `dill` directly here would make more annoying to switch out the library later. Can you use the [`serde`](https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/main/src/orquestra/sdk/_base/serde.py) module? Or add the missing functionality there?